### DISCUSSION TITLE
Is it possible to make NeORG connect to my .dotifles?

### DISCUSSION THREAD

**Discussion by iCraft85** [nvim-neorg/neorg#474] [Q&A]
So with org mode in emacs your able to have your configs be built into the .org files is this possible with neorg? and if so how?

**Comment by max397574** [comment]
this is not possible yet but a planned feature

**Reply by iCraft85** [reply]
ah ok, so is there any estimated time?

**Reply by max397574** [reply]
next few months somewhen 🤷‍♂️

**Comment by refaelsh** [comment]
I am interested in this feature too. Is there a way to track progress while this feature is being worked on? Is there a PR for it? Thanks :-)

**Reply by max397574** [reply]
this is possible since quite a long time already
look into the tangle module
also @iCraft85 if you haven't known yet

**Reply by refaelsh** [reply]
I am sorry, but I've read the Tangle module documentation twice, and I don't see it. Can you please point me?

**Reply by max397574** [reply]
I don't understand what you don't see there
the tangle module is the way to write configs using norg files like you can do with org

**Reply by refaelsh** [reply]
What I mean is like this. Let's say I want to write my config for NixOS in eMacs Org. I would go to the `/etc/nixos/configuration.nix` and put there something like the following:
```org
(org-babel-load-file
 (expand-file-name
  "/home/refaelsh/dotfiles/configuration.org"
  user-emacs-directory))
```
And then I would write my NixOS config in Org using the `/home/refaelsh/dotfiles/configuration.org` instead of writing it in Nix and using the `/etc/nixos/configuration.nix`.

I think that what the OP meant. And me to.

**Reply by max397574** [reply]
well here you'd do the "opposite"
you'd write the config wherever you want and tangle it to where it needs to be

atm there is no nix tool which does extract the code blocks which apparently happens here

and afaict your way also is something nix specific but not sure

**Reply by refaelsh** [reply]
Oh! Now I understand. Thank you very much for your answer.
